addons/rating/models/rating_mixin.py

```diff
@@ -27,9 +27,24 @@ class RatingMixin(models.AbstractModel):
 
     @api.depends('rating_ids.rating', 'rating_ids.consumed')
     def _compute_rating_last_value(self):
+        # Pure SQL instead of calling read_group to allow ordering array_agg
+        if not self.ids:
+            self.rating_last_value = 0
+            return
+        self.env.cr.execute("""
+            SELECT
+                array_agg(rating ORDER BY write_date DESC) AS "ratings",
```

Comment:
Is this faster than a subselect? I wonder what is the cost of `array_agg` vs an indexed scan.
PS: a quick explain analyze on both seems to indicate that it's actually quite similar (without data though).

```suggestion
                (SELECT rating FROM rating_rating r2 WHERE r2.res_id = r.res_id AND r2.res_model = %s AND consumed = true ORDER BY write_date DESC LIMIT 1) AS "ratings",
```

Reply:
Isn't this ordering perf problem fixed by the private `_read_group` added in #84908 ?
(Not 100% sure, but I thought of this change when reading your above comment and diff)